codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/RpcWorker.res

```diff
@@ -162,15 +155,7 @@ module Make = (
         ~eventRouter,
       )
 
-      let eventBatchItems = await eventBatchPromises->Promise.all
-      let parsedQueueItems = switch eventFilters {
-      //Most cases there are no filters so this will be passed throug
-      | None => eventBatchItems
-      | Some(eventFilters) =>
-        //In the case where there are filters, apply them and keep the events that
-        //are needed
-        eventBatchItems->Array.keep(item => item->FetchState.applyFilters(~eventFilters))
-      }
```

Comment:
The only place where the `parsedQueueItems` are used before they are filtered in the new place is for logging the number of batch items (or adding them to the benchmark). So it's safe to move the logic.

Reply:
Nice to have it contained in one module 💪🏼